examples/httpServer.go

```diff
@@ -45,7 +45,7 @@ func main() {
 func testAppHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	ctx := r.Context()
-	creds := ctx.Value(service.CTXKeyCredentials).(credentials.Credentials)
+	creds := ctx.Value(service.CTXKeyCredentials).(*credentials.Credentials)
```

Comment:
I think this would be better as:
```
creds := ctx.Value(service.CTXKeyCredentials).(goidentity.Identity)
```

Reply:
with *cred.. I get  (Printf("%#v", creds)
    creds: &credentials.Credentials{Username:...
Looks good to me.

I tried goidentity.Identity, but I can't seem to find goidentity. Where could I find this package?

Reply:
Oh yes should have let you know the package...
``import goidentity "gopkg.in/jcmturner/goidentity.v3"``

Reply:
Great, seems to work fine. I get the same result:
creds: &credentials.Credentials{Username:...